index.js

```diff
@@ -49,8 +49,10 @@ function walkAST(ast, before, after, options) {
         ast.alternate = walkAST(ast.alternate, before, after, options);
       }
       break;
-    case 'Extends':
     case 'Include':
+      walkAST(ast.block, before, after, options);
```

Comment:
Don't use implicit fall-through please.  Just duplicate the call to walk the filename.